source/onboard/guest-accounts.rst

```diff
@@ -112,29 +99,29 @@ There are `additional permissions </onboard/advanced-permissions.html>`__ in Mat
 Guest identification
 ---------------------
 
-Guests are identified with a **Guest** badge. This badge is visible in various places on the interface and mobile apps, such as on a guest’s profile and next to their name on user lists, including @mentions. When guests are added to a channel, a system message informs other channel members that the added user is a guest.
+Guests are identified with a **Guest** badge unless your system admin has `disabled guest badges </configure/authentication-configuration-settings.html#guest-access>`__. When visible, this badge is visible in various places in the Mattermost interface, such as on a guest’s profile and next to their name on user lists, including @mentions. When guests are added to a channel, a system message notifies other channel members that the added user is a guest.
```

Comment:
The last sentence (the system message) is only true if the setting is on. I think it is not clear the way it is written right now, but 0/5.